### PR TITLE
Config with pure python, plus pypy

### PR DESCRIPTION
**Pull request by mauritsvanrees**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,58 +1,79 @@
...
-basepython =
-    python3.6
+basepython = python3
+skip_install = false
+extras =
+    docs
+commands_pre =
 commands =
     sphinx-build -b html -d docs/_build/doctrees docs docs/_build/html
-    sphinx-build -b doctest -d docs/_build/doctrees docs docs/_build/doctest
```
Comment (icemac) [inline tox.ini:48]: This line can be kept by setting `with-sphinx-doctests = true` in `.meta.toml`.

**Review by icemac — CHANGES_REQUESTED** [review]
Please keep running sphinx doctests.

File: tox.ini (hunk 1 above)
Comment (mauritsvanrees) [reply tox.ini:48]: Done.
I needed to add `zope.app.wsgi` to the `docs` extra to fix those tests.

**Review by icemac — APPROVED** [review]

**Comment by icemac** [comment]
Thank you for this PR. 😃

**Pull request merged**